### ISSUE TITLE
How to improve computecanada bandwidth ?

### ISSUE THREAD

**Issue by ltetrel** [neurolibre/neurolibre-binderhub#11]
How to upgrade or improve the internet speed on the server ?
It is not possible to have 150KB/s if we want to serve a binderhub

**Comment by agahkarakuzu** [comment]
I think this also relates to neurolibre/neurolibre-binderhub_copy#1? 150KB/s should be decent for user interaction, but I see that it can easily bottleneck user-image pulling issue. Let's see if self-owned image registry will do away with it.

**Comment by ltetrel** [comment]
Yes we are agree. Let's keep this issue open if we can have a discussion with compute canada folks and improve the internet. If this is the case most of our technical issues could be easily fixed.
